sql/src/main/java/io/crate/analyze/relations/JoinPairs.java

```diff
@@ -170,4 +170,11 @@ public static void removeOrderByOnOuterRelation(QualifiedName left,
         }
         return outputs;
     }
+
+    /**
+     * Returns true if the given relation is included in the joinPairs
+     */
+    public static boolean containRelation(List<JoinPair> joinPairs, QualifiedName relation) {
```

Comment:
Maybe inline this if there's only a single usage? I'm not sure if the abstraction improves readability